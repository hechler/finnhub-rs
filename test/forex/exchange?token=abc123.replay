{"request":{"url":"https://finnhub.io/api/v1/forex/exchange?token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/forex/exchange?token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400, h3=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"662fcfa00a6d9c75-AMS","cf-request-id":"0ad1c0180100009c750e2d9000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Mon, 21 Jun 2021 19:58:20 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"report_to\":\"cf-nel\",\"max_age\":604800}","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v2?s=A8p2qtBWB6jiMF54gTlJfMgdfxDj8Bcpx1F4mzI35nKUi1kiYQ35VC1pfE8UzP98J%2BMe1ltV43VNurTOB2caiR7miLiTduI2Q0dwNah7U6MMKK6D7j%2FXiQ%3D%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}","server":"cloudflare","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"59","x-ratelimit-reset":"1624305560"},"body":"WyJvYW5kYSIsImZ4Y20iLCJmb3JleC5jb20iLCJwZXBwZXJzdG9uZXVrIiwiaWMgbWFya2V0cyIsIm9jdGFmeCIsImZ4cGlnIiwicGVwcGVyc3RvbmUiLCJmeHBybyIsImljbXRyYWRlciJd"},"format_version":3}