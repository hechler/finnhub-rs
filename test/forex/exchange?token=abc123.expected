[
    "oanda",
    "fxcm",
    "forex.com",
    "pepperstoneuk",
    "ic markets",
    "octafx",
    "fxpig",
    "pepperstone",
    "fxpro",
    "icmtrader",
]